ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.13.yaml

```diff
@@ -45,6 +45,11 @@ tests:
   steps:
     cluster_profile: ibmcloud-qe
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-private
+- as: installer-rehearse-aws
+  interval: 960h
+  steps:
+    cluster_profile: aws
```

Comment:
profile is `aws-qe`

Reply:
Thank you. After comparing with other agents, I have noticed this and have modified it.